### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by VioMNE**

### CODE REVIEW

File: javascript/javascript2/week3/fastestpresser.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const countS = document.getElementById("counter-s");
+const countL = document.getElementById("counter-l");
+const messageForS = document.getElementById("winning-message-s");
+const messageForL = document.getElementById("winning-message-l");
+let confettiGenerator;
+let counterS = 0;
+let counterL = 0;
+let gameEnd = false;
+let gameStart = false;
+
```
Comment (piepear) [inline javascript/javascript2/week3/fastestpresser.js:12]: I really liked this way of programming of having different states of the game. It is usually a very good way of thinking to solve the problems like this. 😄

File: javascript/javascript2/week3/warmup.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  setTimeout(callback, delay * 1000);
+}
+runAfterDelay(4, function () {
+  console.log("Should be logged after 4 sec");
+});
+
+//Check if we have double clicked on the page.
+
+window.addEventListener("dblclick", () => console.log("Double click!"));
+
```
Comment (piepear) [inline javascript/javascript2/week3/warmup.js:69]: I think the point of this exercise was to define the double click as two clicks with 0.5 secs in between. you can easily achieve it with an if statement and a set time out.

File: javascript/javascript2/week3/warmup.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+function logFunnyJoke() {
+  console.log("That was funny");
+}
+function logBadJoke() {
+  console.log("What was that??? Not funny at all!!!");
+}
+
+jokeCreator(true, logFunnyJoke, logBadJoke);
```
Comment (piepear) [inline javascript/javascript2/week3/warmup.js:87]: YAY! Great job! 👏 👏

Comment (VioMNE) [reply javascript/javascript2/week3/warmup.js:87]: Thank you for your input :)

**Pull request merged**